metrics/notebooks/README.md

```diff
@@ -36,7 +36,11 @@ Example for the comparison of three single user phases (phase identifiers taken
 EXPERIMENT_ID = ["test_experiment1", "test_experiment2"]
 EXPERIMENT_START_TIME = ["2023-04-30T12:12:12.000000000Z", "2023-04-30T14:14:14.000000000Z"]
 EVENT_IDS = ["single_user_1", "single_user_2", "single_user_3"]
-DUCKDB_PATH = "./telemetry"
 ```
 
-You can determine the available experiment identifiers and start times by using the ``listExperiments.ipynb`` notebook.
```

Comment:
This seems to revert changes merged to list the experiments in the database, is it intentional?

Reply:
That was not intentional, probably happened when merging main. Fixed!

Reply:
It seems the changes are still present?

Reply:
The last PR still removes the DUCKDB_PATH variable, which I think is still not intended?

Reply:
That is actually intended since DUCKDB_PATH moved into constant.py and thus would not need to be set in the prefix of the notebooks.